app/src/main/java/com/github/se/eduverse/ui/others/setting/Setting.kt

```diff
@@ -1,116 +1,238 @@
 package com.github.se.eduverse.ui.others.setting
 
+import androidx.compose.foundation.background
 import androidx.compose.foundation.clickable
 import androidx.compose.foundation.layout.*
 import androidx.compose.material.icons.Icons
-import androidx.compose.material.icons.filled.ArrowDropDown
+import androidx.compose.material.icons.automirrored.filled.ArrowBack
+import androidx.compose.material.icons.filled.*
 import androidx.compose.material3.*
 import androidx.compose.runtime.*
 import androidx.compose.ui.Alignment
 import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.vector.ImageVector
 import androidx.compose.ui.platform.testTag
+import androidx.compose.ui.text.style.TextOverflow
+import androidx.compose.ui.unit.Dp
 import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
 import com.github.se.eduverse.ui.navigation.NavigationActions
+import com.github.se.eduverse.ui.navigation.Screen
+import com.google.firebase.auth.FirebaseAuth
 
+// Define theme colors
+private val HeaderColor = Color(0xFF4bace5)
+private val SecondaryColor = Color(0xFF217384)
+private val LightBackgroundColor = Color.White
```

Comment:
I don't understand the use of LightBackgroundColor. I feel like it won't change anything in comparison to having a column with default parameter, except that it won't be adapted when the app is put into dark mode